src/gen/umple/dontdelete.txt

```diff
@@ -0,0 +1,3 @@
+This file exists to keep the directory always present in git and thus IDEs can mark as source directories. 
```

Comment:
Never heard of it! It's a matter of convention apparently, and I think I like yours better. I'll change the name.